### ISSUE TITLE
Indentation sniff triggers falsely on switch cases that return arrays. (2.4.0 version)

### ISSUE THREAD

**Issue by msteltenpool** [squizlabs/PHP_CodeSniffer#785]
Squiz_Sniffs_CSS_IndentationSniff is triggered on the following piece of code, narrowed it down to '[ ]' being the problem (simplified example):

```php
switch ($test) {
    case 0:
        return ['array'];
    case 1:
        return ['array'];
}
```

Outcome Line indented incorrectly; expected 8 spaces, found 4 (on the 'case 1:' line). Problem dissapears if we remove the '[ ]';

**Comment by derrabus** [comment]
Same as #786

**Comment by gsherwood** [comment]
There is more info on the other idea, so I'll close this one. It's the exact same issue though.
